### PR TITLE
Log info about the app version and build mode at the app's start [ECR-3150]

### PR DESCRIPTION
**Pull request by skletsun**
## Overview

App's version and build mode are logged with level `INFO` at the app start.

---
See: https://jira.bf.local/browse/ECR-3150

### Definition of Done

- [ ] There are no TODOs left in the code
- [ ] Change is covered by automated [tests](https://github.com/exonum/exonum-java-binding/blob/master/CONTRIBUTING.md#tests)
- [ ] The [coding guidelines](https://github.com/exonum/exonum-java-binding/blob/master/CONTRIBUTING.md#the-code-style) are followed
- [ ] Public API has Javadoc
- [ ] Method preconditions are checked and documented in the Javadoc of the method
- [ ] Changelog is updated if needed (in case of notable or breaking changes)
- [ ] The continuous integration build passes

### CODE REVIEW

File: exonum-java-binding/core/rust/src/runtime/library_loader.rs [hunk 1]
```diff
@@ -23,8 +23,8 @@ pub extern "system" fn Java_com_exonum_binding_util_LibraryLoader_nativeGetLibra
     env.new_string(get_lib_version()).unwrap().into_inner()
 }
 
-// Returns the exact value of the `version` field from the library's Cargo.toml configuration file.
-fn get_lib_version() -> &'static str {
+/// Returns the exact value of the `version` field from the library's Cargo.toml configuration file.
+pub fn get_lib_version() -> &'static str {
```
Comment (dmitry-timofeev) [inline exonum-java-binding/core/rust/src/runtime/library_loader.rs]: Shall it be moved to `utils` for exonum-java to depend on it :thinking: ?

File: exonum-java-binding/core/rust/exonum-java/src/main.rs [hunk 2]
```diff
@@ -19,17 +19,39 @@ extern crate exonum_btc_anchoring;
...
     builder.run()
 }
+
+// Prints info about version and build mode of started app to the STDOUT.
+fn log_app_metadata() {
+    let version = get_lib_version();
+    let build_type = if cfg!(debug_assertions) {
+        "DEBUG"
+    } else {
+        "RELEASE"
```
Comment (dmitry-timofeev) [inline exonum-java-binding/core/rust/exonum-java/src/main.rs]: I'd suggest lowercase.

File: exonum-java-binding/core/rust/src/runtime/library_loader.rs (hunk 1 above)
Comment (skletsun) [reply exonum-java-binding/core/rust/src/runtime/library_loader.rs]: I was also thinking about this but didn't end up with a name for a module except "misc", "common", "other" etc 🙃

File: exonum-java-binding/core/rust/exonum-java/src/main.rs (hunk 2 above)
Comment (skletsun) [reply exonum-java-binding/core/rust/exonum-java/src/main.rs]: Done

**Pull request merged**